user/incus/files/incus

```diff
@@ -0,0 +1,5 @@
+# incus service
+
+type = process
+command = /usr/bin/incusd --group incus-admin --syslog
+depends-on = network.target
```

Comment:
network.target is supplementary, you also need local.target

Reply:
done